tests/suites/test_suite_ssl.function

```diff
@@ -1755,3 +2010,65 @@ void ssl_session_serialize_version_check( int corrupt_major,
 
 }
 /* END_CASE */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_X509_CRT_PARSE_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_RSA_C:MBEDTLS_ECDSA_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
+void test_mbedtls_endpoint_sanity(int endpoint_type)
```

Comment:
```suggestion
void test_mbedtls_endpoint_sanity( int endpoint_type )
```

Reply:
Also, please don't prefix the function name with `test`, since, after generating the test suite .c file, it will become `test_test_`.